yaml/xyz/openbmc_project/LID/Validate.interface.yaml

```diff
@@ -0,0 +1,32 @@
+description: >
+    An interface to validate marker LID before starting inband code update.
+methods:
+    - name: Validate
+      descripton: >
+          Validate file in staging directory given the lid value 7.
```

Comment:
What does "lid value 7" signify?

Reply:
it's the lid marker value that signifies that validation is required for this in band update, and I assume in some cases the validation is unnecessary. should I reword the description?